README.md

```diff
@@ -21,6 +21,16 @@ See [the SQL developer tutorial] to develop an app with C#, Java, Node.js, PHP,
 
 <img src="https://github.com/Microsoft/vscode-mssql/raw/main/images/mssql-demo.gif" alt="demo" style="width:480px;"/>
 
+## Version 1.13.0
+* Release date: February 24, 2022
+* Release status: GA
+
+## What's new in 1.13.0
+* Adds object explorer table integration for SQL bindings with Azure Functions
+* Updated SqlParser to fix some intellisense issues
+* Fix decimal datatype handling
```

Comment:
For fixes it's generally nice to link the issue as well. For this one we don't have a reported issue in this repo so let's just link the ADS one 

Fixes [#275](https://github.com/microsoft/azuredatastudio/issues/275) - Error handling large decimal values